### DISCUSSION TITLE
Lesson 12 getting this error during delpoying contract on rinkeby network

### DISCUSSION THREAD

**Discussion by avikat** [smartcontractkit/full-blockchain-solidity-course-py#1158] [Q&A]
Hi Community, Pleas ehelp me with this error

KeyError: 'OpenZeppelin/openzeppelin-contracts@4.1.0/Address'.

**Comment by n4n0b1t3** [comment]
Please [have a look at this post](https://github.com/smartcontractkit/full-blockchain-solidity-course-py/discussions/980) and update your question in a way, that others can actually help you.

**Reply by cromewar** [reply]
Yes please we will need more info on this @avikat
